frontend/packages/topology/src/elements/BaseGraph.ts

```diff
@@ -118,7 +138,7 @@ export default class BaseGraph<E extends GraphModel = GraphModel, D = any> exten
   }
 
   setScale(scale: number): void {
-    this.scale = scale;
+    this.scale = Math.max(Math.min(scale, this.maxScale), this.minScale);
   }
```

Comment:
Thought we decided that a user should be able to set the scale to anything.
Currently if we use the `fit to screen` option on a very spread out visualization, panning the graph will reset the scale because the behavior is using `setScale`.